packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -18,13 +18,56 @@ declare global {
     }
 }
 
-export type AxisPositioningMode = "uncontrolled" | "locktodefault" | "dynamic";
-export type AxisScalingMode = "anchor" | "fill" | "content";
-export type GbcrUsage = "always" | "never" | "default";
-export type HorizontalPosition = "start" | "end" | "left" | "right" | "unset";
-export type VerticalPosition = "top" | "bottom" | "unset";
```

Comment:
Is there a reason for making the change to having these be enums? Enums don't compress whatsoever and types provide type safety. I'd prefer to keep these as types unless there is an explicit reason to change it over to an enum.

Reply:
types good, enum bad...converted to types

Reply:
made me laugh during a meeting 😆